lib/MusicBrainz/Server/ControllerUtils/JSON.pm

```diff
@@ -0,0 +1,32 @@
+package MusicBrainz::Server::ControllerUtils::JSON;
+
+use base 'Exporter';
+
+our @EXPORT_OK = qw(
+    serialize_pager
+);
+
+sub serialize_pager {
+    my ($pager) = @_;
+
+    return {
+        current_page => $pager->current_page + 0,
```

Comment:
The `+0` is needed so that Perl sees it as a number, and the JSON encoder outputs e.g. `3` instead of `"3"`.